config/default.py

```diff
@@ -84,16 +84,17 @@
 JOB_SCHEDULING_ENABLED = True
 
 # See http://apscheduler.readthedocs.io/en/latest/modules/triggers/cron.html for supported schedule formats.
```

Comment:
Since the URL link doesn't mention Python dictionaries, would it be OK to include an "e.g." sample value? The existing fill-ins were annoying noise so far as runtime utility goes, but they helped meet the default config's documentary goal.

Reply:
Oh yes, sample value in a comment is a good idea.

Reply:
Done.